Configs/.local/lib/hyde/rofi.websearch.sh

```
@@ -88,12 +88,13 @@ handle_query() {
     fi
     write_to_top "$cached_search_dir/$site.txt" "$query"
     write_to_top "$cached_search_dir/recent.sites" "$site | ${SITES[$site]}"
+    query_url=$(echo -n ${SITES[$site]} | sed "s/%s/$query/")
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)